prog/test/verify_purge.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "net/ssh"
+
+class Prog::Test::VerifyPurge < Prog::Test::Base
+  def self.assemble
+    Strand.create_with_id(
+      prog: "Test::VerifyPurge",
+      label: "start",
+      stack: [{}]
+    )
+  end
+
+  label def start
+    verify_vm_dir_purged
+    verify_storage_files_purged
+    verify_spdk_artifacts_purged
```

Comment:
You could spread them across 3 labels instead of making 3 method calls in the same label.